content_api/app/settings.py

```diff
@@ -47,6 +47,7 @@ def env(variable, fallback_value=None):
 OAUTH2_TOKEN_URL = '/token'
 OAUTH2_SCOPES = ['content_api']
 BCRYPT_GENSALT_WORK_FACTOR = 12
+PUBLIC_RESOURCES = ['publish', 'assets', 'clients', 'users']
```

Comment:
From experience, that "temporary" change might hang around for quite some time. :)
IMO it's better to add instructions on how to create a new API client and create the latter during the deployment.